src/csharp/Grpc.IntegrationTesting/ExternalDnsClientServerTest.cs

```diff
@@ -0,0 +1,74 @@
+#region Copyright notice and license
+
+// Copyright 2019 gRPC authors.
```

Comment:
Actually this is the old Copyright format. Use of "The gRPC Authors" instead of "gRPC authors." is now recommended.